src/CircleMenu/CircleMenu.less

```diff
@@ -0,0 +1,17 @@
+.circleContainer {
+  border: dashed 1px;
+  border-radius: 50%;
+  position: relative;
+  z-index: 100;
+  background: -moz-radial-gradient(center, ellipse cover, rgba(0,0,0,0) 0%, rgba(200, 199, 58, 0.65) 100%);
+  background: -webkit-radial-gradient(center, ellipse cover, rgba(0,0,0,0) 0%,rgba(200, 199, 58, 0.65) 100%);
+  background: radial-gradient(ellipse at center, rgba(0,0,0,0) 0%,rgba(200, 199, 58, 0.65) 100%);
```

Comment:
Not sure, if possible, but it may be useful to have the color configurable via less modify variables.

Reply:
That'd be nice indeed.

Reply:
You're right.
https://github.com/terrestris/react-geo/pull/218/commits/16c366f40a7f4cd1668dfc07f83261b235078dbc
introduces a less-variable `@circle-menu-background-color` and the missing `passThroughProps` so you can set the style where you create the CircleMenu.

Btw:
Switching to styled components could solve these and other issues like #225 and would remove some more packages from the dependencies.